server/routes/middleware.js

```diff
@@ -59,4 +60,28 @@ const hasUserSignedBody = async (req, res, next) => {
   next()
 }
 
-module.exports = { partialReqCheck, reqCheck, checkExistence, hasUserSignedBody }
+const validateID = async (req, res, next) => {
```

Comment:
Why is this function here? It is not used in server

Reply:
removed :pray